web/wp-content/themes/dctx-v2/template-parts/content-blocks/block-wds-recent-posts.php

```diff
@@ -0,0 +1,81 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-5xit6c.php, but found phpcs-scan-5XIT6c.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).